db/socialcredit.py

```diff
@@ -0,0 +1,36 @@
+from typing import List, Optional
+
+from sqlalchemy import Integer
+from sqlalchemy.orm import column_property
+from sqlalchemy.schema import Column
+
+from .base import Base
+
+
```

Comment:
This is pretty much the way you setup tables with SQLAlchemy.

1. Inherit from Base
2. Define columns
3. Profit

`column_property` is a way to virtually create a column that exists in the programming level and not the internal db level. Useful for this case when total score is calculated. Less updates like this but slightly more computational overhead